hathor/cli/side_dag.py

```diff
@@ -205,104 +195,82 @@ def is_option(arg_: str) -> bool:
     return hathor_node_argv, side_dag_argv
 
 
-def _run_side_dag_node(argv: list[str], *, hathor_node_process: Process, conn: 'Connection') -> None:
+def _run_monitor_process(*processes: Process) -> None:
```

Comment:
Maybe rename the main process proctitle to `hathor-core: monitor`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1079/commits/e965a13f46986b636960070abbf4693bea9fc5c7

Reply:
Ok! Thanks!